CredentialCardField.vue

```diff
@@ -144,10 +133,10 @@ export default {
   },
   computed: {
     showIcon() {
-      return this.$q.iconSet.credentialCardField.showField;
+      return 'fas fa-eye-slash';
     },
     hideIcon() {
-      return this.$q.iconSet.credentialCardField.hideField;
+      return 'fas fa-eye';
```

Comment:
We need to use the `$q.iconSet` stuff (or similar) for this so we can use the material icons as well.

Reply:
Yeah I am looking into that now and trying to get it to work.

Reply:
@JoshDunnDev in credentialsMixin I missed an icon if you did not get it would be good to change this line: 

https://github.com/digitalbazaar/bedrock-vue-credential-card/blob/q1-css-fixes/credentialMixin.js#L44

Reply:
omg so this is all my fault when I was refactoring I changed the mixin to set credentialCardField and it should be credentialCard in the mixin. I missed the one word difference between the 2 objects. so wow... I did a really bad job on this.

Reply:
this.$q.icon.name is not returning fontawesome anymore it returns font-awesome with a version flag too. anyways I think you are fixing this.